CONTRIBUTING.md

```diff
@@ -37,17 +44,23 @@ This is a rough outline of what a contributor's workflow looks like:
 
 Thanks for your contributions!
 
+### Coding Style
+
+CoreOS projects written in Go follow a set of style guidelines that we've documented
```

Comment:
Doesn't seem necessary to link to the Go style guide when the entire projects in Rust.

Reply:
I just copied this from the template project repo without reading it. I'll update it.

Reply:
actually do we have a convenient place to link to for our rust coding style or should I just delete this section? 

/cc @crawford

Reply:
We can just recommend that rustfmt be used.